app-localize-behavior.html

```diff
@@ -194,6 +195,10 @@
     }
   },
 
+  ready() {
+    this.fire('app-localize-behavior-created', this);
```

Comment:
Yes. I need it to keep a list of all the instances that were created. I use that to inform each of them about a change in the language. So adding this event supports my use case without requiring app-localize-behavior to manage that list.